app/components/mark_complete_component/view.html.erb

```diff
@@ -1,4 +1,5 @@
 <% if @generate_form %>
+  <!-- TODO: find out which test is passing in nil -->
```

Comment:
https://github.com/alphagov/forms-admin/blob/a2a64e3ffc494f069b0429bdddef831e90947b36/spec/components/mark_complete_component/view_spec.rb#L12-L26

Reply:
Hmm I can't remember why this was a problem